native_client/deepspeech.h

```diff
@@ -128,8 +98,77 @@ namespace DeepSpeech
       char* stt(const short* aBuffer,
                 unsigned int aBufferSize,
                 int aSampleRate);
+
+      /**
+       * @brief Setup a context used for performing streaming inference.
+       *        the context pointer returned by this function can then be passed
+       *        to {@link feedAudioContent()} and {@link finishStream()}.
+       *
+       * @param aPreAllocFrames Number of timestep frames to reserve. One timestep
+       *                        is equivalent to two window lenghts (50ms), so
+       *                        by default we reserve enough frames for 3 seconds
+       *                        of audio.
+       * @param aSampleRate The sample-rate of the audio signal.
+       *
+       * @return A context pointer that represents the streaming state.
```

Comment:
We should mention it might return a null pointer

Reply:
Done.